packages/loot-design/src/components/modals/SelectLinkedAccounts.js

```diff
@@ -1,254 +1,200 @@
-                          }}
-                          value={selectedImportAccountId}
-                        />
-                      </View>
-
-                      <View>
-                        <Strong>Actual Budget Account:</Strong>
-                        <CustomSelect
-                          options={actualAccountsToSelect.map(account => [
-                            account.id,
-                            account.name,
-                          ])}
-                          onChange={val => {
-                            setSelectedAccountId(val);
-                          }}
-                          value={selectedAccountId}
-                        />
-                      </View>
-
-                      <Button
-                        primary
-                        style={{
-                          padding: '10px',
-                          fontSize: 15,
-                          margin: 10,
-                        }}
-                        onClick={addToChosenAccounts}
-                      >
-                        Link account &rarr;
-                      </Button>
-                    </View>
-                  ) : (
-                    ''
-                  )}
-                  {chosenAccounts.map(chosenAccount => {
-                    const { chosenImportedAccountId, chosenActualAccountId } =
-                      chosenAccount;
-                    const importedAccount = importedAccounts.find(
-                      acc => acc.account_id === chosenImportedAccountId,
-                    );
-                    const actualAccount = [
-                      addAccountOption,
-                      ...actualAccounts,
-                    ].find(acc => acc.id === chosenActualAccountId);
-
-                    return (
-                      <View
-                        key={chosenImportedAccountId}
-                        style={{
-                          flexDirection: 'row',
-                          justifyContent: 'flex-center',
-                          marginTop: 30,
-                        }}
-                      >
-                        {importedAccount.name} &rarr; {actualAccount.name}
-                        <Button
-                          primary
-                          style={{
-                            padding: '10px',
-                            fontSize: 15,
-                            margin: 10,
-                          }}
-                          onClick={() => removeChoose(chosenAccount)}
-                        >
-                          Remove &rarr;
-                        </Button>
-                      </View>
-                    );
-                  })}
+            <TableHeader
+              headers={[
+                { name: 'External account name', width: 200 },
+                { name: 'Local account name', width: 'flex' },
+                { name: 'Actions', width: 'flex' },
```

Comment:
I don’t think this column needs a title (and it looks a bit weird to have the title on the left and the button on the right)

Reply:
Makes sense: https://github.com/actualbudget/actual/pull/792/commits/9fd061cdeb42a1430d04fe689a9703a30add07b7